style/testing/avoid_let_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe SomeClass do
+  describe "#let" do
+    let(:user) { User.new(name: "John") }
+
+    context "avoid using" do
+      it "uses lets" do
+        # BAD
+        user.name = "Jane"
+        result = SomeClass.new(user).run
+
+        expect(result).to be_present
+      end
+
+      it "does not use lets" do
+        # GOOD
+        user = build_user(name: "Jane")
```

Comment:
is `build_user` a method that would go into this spec somewhere? is it factory_girl? something else?